Mathlib/MeasureTheory/Function/ConditionalExpectation/Basic.lean

```diff
@@ -420,4 +420,48 @@ theorem tendsto_condexp_unique (fs gs : ℕ → α → F') (f g : α → F')
   exact tendsto_nhds_unique_of_eventuallyEq hcond_gs hcond_fs (eventually_of_forall hn_eq)
 #align measure_theory.tendsto_condexp_unique MeasureTheory.tendsto_condexp_unique
 
+open Set
+
```

Comment:
It saves you a single `Set.` in the code. That does not seem worth it. You can event replace that `Set.indicator A` by `A.indicator`.
```suggestion
```